include/smack/SmackRep.h

```diff
@@ -67,6 +67,10 @@ class SmackRep {
 
   vector<Region> memoryRegions;
   const llvm::DataLayout* targetData;
+
+  // TODO Make this width a parameter to generate bitvector-based code.
+  unsigned width;
```

Comment:
This field denotes the width of pointers, and should be named accordingly. Furthermore since the width of pointers need not be equal to the width of `size_t`, nor the maximum width of an integer type, this field should not be used in those other situations.